cmd/agent/api/server.go

```diff
@@ -140,6 +140,7 @@ func StartServer(configService *remoteconfig.Service) error {
 		TLSConfig: &tls.Config{
 			Certificates: []tls.Certificate{*tlsKeyPair},
 			NextProtos:   []string{"h2"},
+			MinVersion:   tls.VersionTLS13,
```

Comment:
should we use the config option `min_tls_version` to set the minVersion
https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config.go#L368

also should we remove the envvar `DD_FORCE_TLS_12` introduced by this PR https://github.com/DataDog/datadog-agent/pull/7546

Reply:
I'm not sure if it's the same context but just in case `min_tls_version` now defaults to `tlsv1.2` ([see PR](https://github.com/DataDog/datadog-agent/pull/13867))